rids.py

```diff
@@ -51,9 +51,11 @@ def retrieve_bad_ips():
         for line in ioc_data.readlines():
           line = line.strip().decode('utf-8')
           if not line: continue
-          bad_ip = ipaddress.ip_address(line)
-          # TODO we need to handle the possibility that an IPaddr appears in more than one list
-          bad_ips[bad_ip] = source['name']
+          bad_ip = str(ipaddress.ip_address(line))
```

Comment:
Please put the file parsing in a separate function. Make it add to a Rules object kind of thing.

Reply:
I've started work on that in a separate branch already, do you mind if it doesn't block this PR?

Reply:
added a TODO